plugin/storage/es/factory.go

```diff
@@ -317,9 +310,35 @@ func (f *Factory) CreateSamplingStore(maxBuckets int) (samplingstore.Store, erro
 		Lookback:               f.primaryConfig.AdaptiveSamplingLookback,
 		MaxDocCount:            f.primaryConfig.MaxDocCount,
 	})
+
+	if f.primaryConfig.CreateIndexTemplates && !f.primaryConfig.UseILM {
+		mappingBuilder := getMappingBuilder(f.primaryConfig, f.templateBuilder)
+		sampleMapping, err := mappingBuilder.GetSamplingMappings()
+		if err != nil {
+			return nil, err
+		}
+		if err := store.CreateTemplates(sampleMapping); err != nil {
```

Comment:
can we delete CreateTemplates methods and inline it here? It's a one-liner that should not be called from anywhere else.

Reply:
76397788b289326949557c0c7b7a988903e23fb0